src/jit/gentree.h

```diff
@@ -598,9 +598,14 @@ struct GenTree
         return isIndir() || isLclField();
     }
 
-    bool isContainedMemoryOp() const
+    bool isUsedFromMemory() const
     {
-        return (isContained() && isMemoryOp()) || isContainedLclVar() || isContainedSpillTemp();
+        return ((isContained() && (isMemoryOp() || isContainedLclVar())) || isUsedFromSpillTemp());
```

Comment:
No, not really. I guess conceptually I was grouping the "real" contained things together. It is probably better to leave them separate.